yarn.lock

```diff
@@ -274,6 +274,10 @@ array-flatten@1.1.1:
   version "1.1.1"
   resolved "https://registry.yarnpkg.com/array-flatten/-/array-flatten-1.1.1.tgz#9a5f699051b1e7073328f2a008968b64ea2955d2"
 
+array-iterate@^1.0.0:
```

Comment:
how is this related to adding a `.coafile` ?

Reply:
Ok will remove it.